INSTALL.asciidoc

```diff
@@ -2,19 +2,30 @@
 pragmatic development environment on openSUSE
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-1.  install openQA package to get all the runtime dependencies
+1.  https://github.com/os-autoinst/openQA/blob/master/docs/Installing.asciidoc#installation[install the openQA package] to get all the runtime dependencies
 +
 ---------------------
 sudo zypper in openQA
 ---------------------
++
+Alternatively installing the following packages will satisfy all dependencies at time of writing
++
+-----------------------------------------------------------------------
+zypper qemu tesseract optipng opencv tesseract-ocr sqlite3 mumble     \
+       libtheora-devel opencv-devel fftw3-devel libsndfile-devel      \
+       pkg-config libtool autoconf automake gcc-c++ make perl-JSON    \
+       perl-Mojolicious perl-Mojolicious-Plugin-AssetPack             \
+       perl-IPC-System-Simple perl-Exception-Class perl-Test-ANSIColor\
+       perl-Test-Compile perl-Test-More perl-Test-Simple perl-XML     \
+       perl-carp perl-Class-Accessor perl-Net-SSH2 perl-Crypt-DES
+-----------------------------------------------------------------------
```

Comment:
While I value your approach to update the documentation by adding this explicit list a new update will be necessary pretty soon and I would not recommend to duplicate the requirements in a hand-written document. A better approach would be to rely on the tools to tell the user what's missing, in this case it's autotools. Of course, it would be nice to rewrite the automake approach to something more modern if that works, e.g. either more perlish Makefile stuff or CMake :-)